pvHelpers/proxy_config/pac_parser.py

```diff
@@ -28,13 +28,12 @@ def __parse_pac(self):
         try:
             pacparser.init()
             if downloaded_pac is not None:
-                g_log.debug("pac url is a url: {}".format(self.pac_url))
                 pacparser.parse_pac_string(downloaded_pac)
             else:
-                g_log.debug("pac url is a local file: {}".format(self.pac_url))
-                pacparser.parse_pac_file(self.pac_url)
+                pacparser.parse_pac_file(parse_file_uri(self.pac_url))

```

Comment:
I think you should attempt to find the scheme before tryin to download the file. could be done w a regex accepting `file` `http` `https`. no scheme and `file` means it's local. It's good idea to unquote it as u r ( I guess you could just directly use `urlparse.unquote` ?)

Reply:
current fall back logic works relatively well.

So, i don't want to make any changes since it requires new testing.

`parse_file_uri` is supposed to behave as if nothing happens if there is no `file://`.